ui/pages/confirm-token-transaction-base/confirm-token-transaction-base.js

```diff
@@ -120,7 +123,18 @@ export default function ConfirmTokenTransactionBase({
     assetStandard === TokenStandard.ERC721 ||
     assetStandard === TokenStandard.ERC1155
   ) {
-    title = assetName || getTitleTokenDescription();
+    if (assetName) {
+      const isExistingNft = nfts?.find(
+        ({ address, tokenId: _tokenId }) =>
+          isEqualCaseInsensitive(tokenAddress, address) && _tokenId === tokenId,
+      );
+      title = assetName;
+      if (!isExistingNft && assetStandard === TokenStandard.ERC721) {
+        title = `${title} #${tokenId}`;
+      }
```

Comment:
maybe 
```suggestion
      title = !isExistingNft && assetStandard === TokenStandard.ERC721
        ? `${title} #${tokenId}`
        : assetName;
```

Reply:
<img width="903" alt="Screen Shot 2023-05-12 at 3 07 35 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/a6e5be03-a0f3-48e3-b182-2dd667c6f444">

oh no, I think my suggestion was not lint-free. sorry about that